### PR TITLE
mpc: update page

### LINKED ISSUE #12318: Page modification request: mpc
### Command description

The mpc page currently says

```
mpc [+-]seconds
```

but it should say

```
mpc seek [+-]seconds
```

### Command details

_No response_

### Documentation

https://www.musicpd.org/doc/mpc/html/

### Platform

Linux

### VCS repository link (e.g. GitHub, GitLab)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by acuteenvy**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

Closes #12318.

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/mpc.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # mpc
 
 > Music Player Client.
-> Program for controlling the Music Player Daemon (MPD).
-> More information: <https://www.musicpd.org/clients/mpc>.
+> A program for controlling the Music Player Daemon (MPD).
```
Comment (Managor) [inline pages/common/mpc.md:4]: If I'm remembering correctly, we're trying to get rid of declaring that these are programs.
```suggestion
> Control the Music Player Daemon (MPD).
```

**Review by Managor — APPROVED** [review]
Yep, looks good to me.

**Comment by Managor** [comment]
What?

**Pull request merged**